srv/modules/runners/ui_rgw.py

```diff
@@ -97,8 +97,16 @@ def _urls(self):
                 self.credentials['urls'].append(cached[minion]['rgw_endpoint'])
                 return
 
+        port = '7480'  # civetweb default port
+        if os.path.exists("/srv/salt/ceph/configuration/files/ceph.conf.rgw"):
+            with open("/srv/salt/ceph/configuration/files/ceph.conf.rgw") as rgw_conf_file:
+                for line in rgw_conf_file:
+                    if line:
```

Comment:
I would assume for line in file would already not return None, if you're checking for emptiness (probably not necessary as the re.search would not return?)